site/content/api/Tenant-API.md

```diff
@@ -135,15 +139,27 @@ The response message's *status* property may contain the following codes:
 | Code  | Description |
 | :---- | :---------- |
 | *204* | No Content, the tenant has been updated successfully. |
-| *404* | Not Found, there is no tenant identified with the given *tenant-id*. |
+| *404* | Not Found, there is no tenant identified with the given *tenant_id*. |
 
 For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
 
 ## Remove Tenant
 
-Clients use this command to *remove* all information about a tenant from Hono. Once a tenant is removed, clients can no longer use Hono to interact with any device of the tenant nor can they consume any more data produced by the devices of the tenant.
+Clients use this command to *remove* all information about a tenant from Hono.
+
+Once a tenant is removed:
+
+* Consumers of the removed tenant can no longer connect to Hono to receive device data.
+* Consumers of the removed tenant can no longer interact with Hono APIs (e.g. the *Device Management API*).
+* Devices of the removed tenant will not longer receive any data from Hono.
+* Devices of the removed tenant are unable to upload data.
```

Comment:
... unable to send data downstream.

Reply:
well, from the device's perspective, they are uploading data to the protocol adapter. I would stick with the original text ...

Reply:
Ok, makes sense in this phrase!
So, just ignore this comment please...